### PR TITLE
Update adsc.go

### PR DESCRIPTION
**Pull request by jessicegao**
**Please provide a description of this PR:**

fix https://github.com/istio/istio/issues/48414

### CODE REVIEW

File: pkg/adsc/adsc.go
Comment (wulianglongrd) [inline pkg/adsc/adsc.go:947]: It looks like the same effect as moving `a.InitialLoad = time.Since(a.watchTime)` here.

In addition, it may not be safe: when sending LDS or RDS encounters an unexpected situation (note that we do not handle sending errors), there may never be a chance to resend LDS?

Comment (wulianglongrd) [reply pkg/adsc/adsc.go:947]: > I am not sure what do you mean.

see L980 https://github.com/istio/istio/pull/49200/files#diff-b5f677f06a05070210449a3f61f748b01824b494d5cf4116f88e50606c621d47R980

Comment (jessicegao) [reply pkg/adsc/adsc.go:947]: should i do the error handling in a following work, the framework need some refactor

Comment (wulianglongrd) [reply pkg/adsc/adsc.go:947]: Please wait for the maintainer review and ignore my comment.

**Review by howardjohn — APPROVED** [review]

**Comment by jessicegao** [comment]
I will try to do the error handling

**Pull request merged**
